src/reducers/optionReducer.ts

```diff
@@ -16,8 +17,11 @@ type OptionState = {
   networkOptions: Options;
 };
 
+const initialNodeLabels: NodeLabel[] = Object.entries(INITIAL_LABEL_MAPPINGS).map(([type, field]) => 
+  ({ type: type.split('_')[0].toLowerCase(), field }));
```

Comment:
I don't think there is a good reason to include '_LABEL' in the mapping, and just makes it error prone. For example, if I have a node with the label Operating_System, it will only match OPERATING_LABLE instead of OPERATING_SYSTEM_LABLE.